src/index.ts

```diff
@@ -404,14 +404,15 @@ export function cast(field: Field, value: string | null): any {
     case 'TIME':
     case 'DATETIME':
     case 'TIMESTAMP':
+    case 'GEOMETRY':
+      return value
```

Comment:
GEOMETRY should be handled with `uint8Array` since it's binary.

https://github.com/planetscale/database-js/issues/98